opencog/atomese/atomese_utils/constants.h

```diff
@@ -14,7 +14,7 @@ const static Handle compressed_value_key = createNode(NODE, "*-AS-MOSES:SchemaCo
 
 const static ValuePtr true_value(createLink(TRUE_LINK));
 const static ValuePtr false_value(createLink(FALSE_LINK));
-extern combo::type_node t_output_type;
+static combo::type_node t_output_type;
```

Comment:
But I still don't understand why it has to be global. Could you add a comment above it to explain?